test/feature/windows/agent_config.json

```diff
@@ -217,7 +217,32 @@
           }
       },
       "metrics_collected": {
-        "emf": { }
+        "emf": { },
```

Comment:
This is going to be odd.. we have a prometheus integ test already that now runs on linux but now this sits with the windows integ tests.
New developers might struggle to understand what sits where.